Sources/CoverageHighlighter/CoverageHighlighter.swift

```diff
@@ -11,6 +11,19 @@ protocol NamedCoverable: Coverable {
     var name: String { get }
 }
 
+public struct Filter {
+    struct FilterFile {
+        let nameSuffix: String
+        let functionPrefix: String?
+    }
+    var filesToFilter: [FilterFile]
+    var filesToExclude: [FilterFile]
+}
+
+extension Filter {
+    var include: [String] { filesToFilter.map { $0.nameSuffix } }
```

Comment:
as this applies to files it would be better to give better names here? I needed to look some time here to understand what this was doing

Reply:
yeah definelty the naming in the whole file is terrible.